Modules/Models/Configuration.swift

```diff
@@ -133,6 +139,20 @@ public struct Configuration: Codable {
     return self.faqURL[language.rawValue] ?? self.faqURL[UserLanguage.english.rawValue]
   }
 
+  /// The Terms Of Use url for the given language. it returns english version if the given
+  /// language is not available.
+  /// Note that the method may still fail in case of missing english version
+  public func termsOfUseURL(for language: UserLanguage) -> URL? {
+    return self.termsOfUseURL[language.rawValue] ?? self.termsOfUseURL[UserLanguage.english.rawValue]
+  }
+
+  /// The Privacy Notice url for the given language. it returns english version if the given
+  /// language is not available.
+  /// Note that the method may still fail in case of missing english version
+  public func privacyNoticeURL(for language: UserLanguage) -> URL? {
+    return self.privacyNoticeURL[language.rawValue] ?? self.privacyNoticeURL[UserLanguage.english.rawValue]
+  }
+
 
```

Comment:
<!--
  0 failure: 
  1 warning:  swiftlint (0.39.2...
  
  
  DangerID: danger-id-Danger;
  File: Modules/Models/Configuration.swift;
  Line: 156;
-->

- :warning: swiftlint (0.39.2): Limit vertical whitespace to a single empty line. Currently 2. (vertical_whitespace)